src/breadcrumb/breadcrumb.component.ts

```diff
@@ -71,6 +80,10 @@ export class Breadcrumb {
 		return this._threshold;
 	}
 
+	ngAfterContentInit() {
```

Comment:
this won't let us un-set the skeleton state ... we'll need something else to update the child skeleton state when the input changes. Also `children` instead of `childs`